packages/expo-dev-client/package.json

```diff
@@ -6,6 +6,7 @@
   "types": "build/DevClient.d.ts",
   "scripts": {
     "build": "expo-module build",
+    "build:plugin": "yarn tsc -p ./plugin/tsconfig.json -w",
```

Comment:
You don't need to do this, running `yarn prepare` already builds the plugin (through `expo-module prepare`).